packages/tokens/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "target": "ES2019",
+    "noEmit": true,
+    "isolatedModules": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "preserveConstEnums": true
```

Comment:
I can't see a ton in the context of the files (since they were mostly all just moved) but if we have any large number of const enums we may want to consider moving to either a frozen object or simply a const object. My understanding is that this "preserves" the const enums so they can be used as values, but in turn that prevents them from compressing, whereas a const object (or using object.freeze) would enable compression.

Reply:
I'm far from a TS expert so I may be wrong here but if my reading of the T[S docs is correct](https://www.typescriptlang.org/tsconfig#references-to-const-enum-members) we cannot be using `const enum` anywhere (if this package builds) as `isolateModules` is set to `true`.

Reply:
This should not be a problem in the context of the `tokens` package as it does not export any `enum`.
The tsconfig has not been manually created, it is what the create-package generates, that's what we have for all FUIR v9 packages.
FWIW, Typescript defaults `preserveConstEnums` to `true` if `isolatedModules` are set to `true`. ([TS docs](https://www.typescriptlang.org/tsconfig#preserveConstEnums))

@hotell - could you please provide us with explanation?